src/main/java/com/mraof/minestuck/event/ServerEventHandler.java

```diff
@@ -154,6 +155,13 @@ public static void onEntityAttack(LivingHurtEvent event)
 					double modifier = PlayerSavedData.getData(player).getEcheladder().getUnderlingDamageModifier();
 					event.setAmount((float) (event.getAmount() * modifier));
 				}
+				
+				Title title = PlayerSavedData.getData(player).getTitle();
+				boolean isVoid = title != null && title.getHeroAspect() == EnumAspect.VOID;
+				if (player.getHeldItemMainhand().getItem() == MSItems.DESOLATOR_MACE && isVoid)
+				{
+					event.getSource().setDamageBypassesArmor();
```

Comment:
I'm a bit wary of modifying the damage source like this, as it is normally set during initialization of a damage source. It's possible that this may cause unexpected behavior in special circumstances. It might be fine most of the time, but to be on the safe side I'd prefer a different method.